src/main/java/gregtech/common/tileentities/generators/GT_MetaTileEntity_MagicalEnergyAbsorber.java

```diff
@@ -109,34 +110,31 @@ public void onPostTick(IGregTechTileEntity aBaseMetaTileEntity, long aTick) {
             // Energyzed node
             if (isThaumcraftLoaded) {
                 try {
+                	int multFactor = 2;
                     World tmpWorld = this.getBaseMetaTileEntity().getWorld();
                     int tmpX = this.getBaseMetaTileEntity().getXCoord();
-                    int tmpY = this.getBaseMetaTileEntity().getYCoord();
+                    int tmpY = this.getBaseMetaTileEntity().getYCoord() + 2; //Add 2 because +1 is the node stabiliser and +2 is the node itself
```

Comment:
These coordinates are of the energy absorber. We have no idea where the node or any thaumcraft parts are. We only tell the TC API that here is something that wants to consume Vis and it pushes it there if in range.